pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -1047,11 +982,17 @@ metadata:
   name: jiva-volume-create-putreplicadeployment-default
 spec:
   meta: |
+    {{- $numbers := mkNumberedSlice .Config.ReplicaCount.value -}}
     id: createputrep
     runNamespace: {{ .TaskResult.createputsvc.jivapodsns }}
     apiVersion: apps/v1
     kind: Deployment
```

Comment:
still in Deployments, there will be names of all the nodes

Reply:
This is the `jiva-volume-create-putreplicadeployment-default` for creating the deployments. How will this affect the node names in patch?